### ISSUE TITLE
Gallery images are missing alt text

### ISSUE THREAD

**Issue by akslay** [az-digital/az_quickstart#1844]
## Problem/Motivation
Gallery images aren't rendering alt text, even when it's provided in the media item. This is an accessibility issue that should be fixed ASAP.

Issue was found by the DRC team on one of their websites:
https://adaptiveathletics.arizona.edu/runnroll

### Describe the bug
Alt text not rendering.

### To Reproduce
1. Create a gallery of images, provide alt text for each image. 
2. View the live page
3. Inspect an image and notice the alt text isn't rendered.

## Proposed resolution
Might be an issue with the twig template (will investigate).

### Expected behavior
Alt text should render when provided.
